integrations/rendertron/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM node:12-slim
```

Comment:
Do we really need our own Dockerfile for rendertron? Do they not provide their own docker image on docker hub that we could directly use?

Reply:
No, they don't provide now. Ref https://github.com/GoogleChrome/rendertron#deploying-using-docker